dev/merge_spark_pr.py

```diff
@@ -85,6 +85,12 @@ def get_json(url):
                 + "dev/merge_spark_pr.py to configure an OAuth token for making authenticated "
                 + "GitHub requests."
             )
+        elif e.code == 401:
```

Comment:
Hm, is 401 always about GITHUB_OAUTH_KEY expiry?

Reply:
In Github rest APIs, 401 stands for `requires authentication`. The token could be invalid with expiry or wrong configuration